README.md

```diff
@@ -37,153 +39,51 @@ For more ways to install the CLI, see [this section](#more-ways-to-install-the-c
+
+See each command's corresponding README for more details, or click on 🔗 to see the related documentation page.
+
+- `cloud-run`:
+  - `flare`: Troubleshoot your issues with [Cloud Run service](src/commands/cloud-run) configuration. [🔗](https://docs.datadoghq.com/serverless/google_cloud_run)
+- `dsyms`:
+  - `upload`: Upload [iOS dSYM files](src/commands/dsyms) for Error Tracking (macOS only). [🔗](https://docs.datadoghq.com/real_user_monitoring/error_tracking/ios/)
+- `flutter-symbols`:
+  - `upload`: Upload [Flutter symbols](src/commands/flutter-symbols) for Error Tracking. [🔗](https://docs.datadoghq.com/real_user_monitoring/error_tracking/flutter/)
+- `git-metadata`:
+  - `upload`: Upload [Git metadata](src/commands/git-metadata) for Source Code Integration. [🔗](https://docs.datadoghq.com/integrations/guide/source-code-integration/)
+- `junit`:
+  - `upload`: Upload [JUnit test reports](src/commands/junit) for Test Visibility. [🔗](https://docs.datadoghq.com/tests/setup/junit_xml/)
+- `lambda`:
+  - `flare`: Troubleshoot your issues with Datadog instrumentation on your [AWS Lambda functions](src/commands/lambda).
+  - `instrument`: Apply Datadog instrumentation to the given [AWS Lambda functions](src/commands/lambda).
+  - `uninstrument`: Revert Datadog instrumentation from the given [AWS Lambda functions](src/commands/lambda).
+- `metric`: Add [metrics](src/commands/metric) to a CI Visibility Pipeline trace or Job span in Datadog. [🔗](https://docs.datadoghq.com/continuous_integration/pipelines/custom_tags_and_metrics/)
+- `react-native`:
+  - `codepush`: Upload [React Native CodePush sourcemaps](src/commands/react-native) for Error Tracking. [🔗](https://docs.datadoghq.com/real_user_monitoring/mobile_and_tv_monitoring/setupcodepush/)
+  - `upload`: Upload [React Native sourcemaps](src/commands/react-native) for Error Tracking. [🔗](https://docs.datadoghq.com/real_user_monitoring/error_tracking/reactnative/)
+  - `xcode`: Upload [React Native sourcemaps](src/commands/react-native) for Error Tracking from the XCode bundle build phase. [🔗](https://docs.datadoghq.com/real_user_monitoring/error_trackingreactnative/)
+- `sarif`:
+  - `upload`: Upload [Static Analysis Results Interchange Format (SARIF)](src/commands/sarif) reports to Datadog. [🔗](https://docs.datadoghq.com/static_analysis/)
+- `sourcemaps`:
+  - `upload`: Upload [JavaScript sourcemaps](src/commands/sourcemaps) for Error Tracking. [🔗](https://docs.datadoghq.com/real_user_monitoring/browser/)
+- `stepfunctions`:
+  - `instrument`: Subscribe [AWS Step Function](src/commands/stepfunctions) log groups to a Datadog Forwarder. [🔗](https://docs.datadoghq.com/serverless/step_functions/installation/tab=datadogcli)
+  - `uninstrument`: Unsubscribe a [AWS Step Function](src/commands/stepfunctions) log group from the specified Datadog Forwarder. [🔗](https://docs.datadoghq.com/serverless/step_functions/installation/tab=datadogcli)
```

Comment:
- `uninstrument`: Uninstrument [AWS Step Function](src/commands/stepfunctions).


Thank you for created this PR and update for us, @Drarig29 !